Add Research page Changes

RitwikSrivastava opened the pull request:
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes #218 

Test URLs:
- Original: https://www.sunstar.com/research/
- Before: https://main--sunstar--hlxsites.hlx.page/_drafts/ritsriva/research1
- After: https://researchpage--sunstar--hlxsites.hlx.page/_drafts/ritsriva/research1


- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

JiangLong2019 commented:
Some comments.

1. Looks like the cards block is broken.
<img width="1257" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/f82f3236-f972-4842-8747-63bcd2630b4c">

2. Scientific Literature content shows different.
<img width="1247" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/39cf537b-93c4-497f-8fcb-64fb357c3949">

3. Text overlap image area lose title and shows different from original site.
<img width="1326" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/b00ab5d6-d132-4d83-9a05-a91e3ad49f1f">

4. Lose R&D Stories content
<img width="1363" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/15799502-0cf7-4f12-83e7-6b10f95f2269">

5. A scroll bar shows in mobile view.
<img width="455" alt="image" src="https://github.com/hlxsites/sunstar/assets/57093988/4b97612c-e3f7-47cd-adc5-f2f32bb3f684">

RitwikSrivastava commented:
> Some comments.

> 2. Scientific Literature content shows different.
I made it consistent with the already available block . and icon not appearing issue i have fixed. 

> 3. Text overlap image area lose title and shows different from original site.
The text is now visible and iam keeping it consitent with the already available hero block instead of creating a variant as discussed.

> 4. Lose R&D Stories content
This needs to be worked upon. It will be a feed of Stories. 

> 5. A scroll bar shows in mobile view.
Fixed. 
@JiangLong2019

sdmcraft commented on blocks/carousel/carousel.css:
```diff
@@ -56,6 +76,24 @@
+  margin-bottom: 1.875rem;
+}
+
+.carousel.small p{
```
Looks like incorrect indentation

sdmcraft commented on blocks/carousel/carousel.css:
```diff
@@ -56,6 +76,24 @@
+}
+
+.carousel.small p{
+  font-size: 14px;
```
Use global variables for font-size from styles.css

sdmcraft commented on blocks/columns/columns.css:
```diff
@@ -451,3 +457,13 @@
   }
 }
+
+@media (max-width: 61rem){
```
Please use mobile first `mid-width` media queries.

sdmcraft commented:
Some comments from UI review

![Screenshot 2023-10-27 at 10 29 17 AM](https://github.com/hlxsites/sunstar/assets/1191451/30374346-d109-4ca1-abf5-9b22a89c33ef)
![Screenshot 2023-10-27 at 10 26 58 AM](https://github.com/hlxsites/sunstar/assets/1191451/8f9f2cad-3fc3-4d85-8bac-e3776f3c3f9e)
![Screenshot 2023-10-27 at 10 26 00 AM](https://github.com/hlxsites/sunstar/assets/1191451/5b9ec216-871c-43dd-8488-3e484a4d64b3)
![Screenshot 2023-10-27 at 10 24 33 AM](https://github.com/hlxsites/sunstar/assets/1191451/5afcfdbd-e26b-4c79-9bbc-c27b249a5d1e)
![Screenshot 2023-10-27 at 10 23 56 AM](https://github.com/hlxsites/sunstar/assets/1191451/afb14e9b-46e5-4989-a07b-db3ec7344464)
![Screenshot 2023-10-27 at 10 21 30 AM](https://github.com/hlxsites/sunstar/assets/1191451/c0b38b89-3c19-4be9-a85c-0325603277df)
![Screenshot 2023-10-27 at 10 19 03 AM](https://github.com/hlxsites/sunstar/assets/1191451/a41de8b1-357c-4275-af48-aa890ccb0a1d)

RitwikSrivastava commented:
@sdmcraft Please re-review. All changes made (margins, image size in mobile views) and also the mobile first media queries.
